simulate_scanning_strategy.py

```diff
@@ -205,26 +214,42 @@ def main():
         # fractions of an arcmin over a time span of one hour)
         time0 = times[0]
         icrs_pos = get_ecliptic_vec(get_body_barycentric(params.planet_name, time0))
-        sat_pos = get_ecliptic_vec(get_body_barycentric("earth", time0))
+        earth_pos = get_ecliptic_vec(get_body_barycentric("earth", time0))
 
         # Move the spacecraft to L2
-        sat_pos = sat_pos * (
-            1.0 + params.earth_L2_distance_km / norm(sat_pos).to("km").value
+        L2_pos = earth_pos * (
+            1.0 + params.earth_L2_distance_km / norm(earth_pos).to("km").value
+        )
+
+        # Creating a circular orbit
+        scann = sim.parameters["scanning_strategy"]
+        R = conversion(scann["radius"], "au")
```

Comment:
This should be added to the `Parameters` class instead of being accessed directly here. Also, `radius_au` would be clearer than `radius`.

Reply:
I would suggest to leave the name "radius". In fact in the toml file you can assign to it a value in km too.

Reply:
Ok. Does it error if one does **not** provide a measure unit? If it does, then it's ok to leave `radius`. If not, it would probably be nicer to have `radius_au`; otherwise the following line in the TOML would look confusing:

```toml
radius = 0.3   # What is the measure unit?
```

If this line raises an error like *Error, no measurement unit specified for 'radius'*, then `radius` is perfect. Otherwise it's better to have `radius_au`:

```toml
radius_au = 0.3   # Much clearer!
```

Of course, one could write

```toml
radius_au = "100000 km"
```

but it would still be clear what is the meaning.

Reply:
Gotcha! I'll change to radius_au.